src/scalper_utils.ts

```diff
@@ -283,22 +283,23 @@ export function getPayerAccount(hedgeSide: 'buy' | 'sell', base: SYMBOL, quote:
   return baseTokenAccount;
 }
 
-export async function cancelTxOpenBookOrders(
+export async function cancelOpenBookOrders(
```

Comment:
Where is this? 'Changing behavior for if cancels fail, do not place new bids'

Reply:
removing the try catch around the transactions. so if there is a failure, it will bubble up and cause the current iteration to fail